import-export-cli/integration/testutils/throttlePolicy_testUtils.go

```diff
@@ -0,0 +1,244 @@
+	uuid := client.GetThrottlePolicyID(t, args.Admin.Username, args.Admin.Password, throttlePolicyName, policyType)
+	policy := client.GetThrottlePolicy(uuid, policyType)
+	client.DeleteThrottlePolicy(uuid, policyType)
+	return ThrottlePolicyStructToMap(policy)
+}
+
+// Validates whether throttling policies are equal
+func ValidatePoliciesEqual(t *testing.T, args *ThrottlePolicyImportExportTestArgs, importedPolicy map[string]interface{}) {
+	exportedPolicy := args.Policy
+	exportedPolicy[policyIDKey] = policyIDReset
+	importedPolicy[policyIDKey] = policyIDReset
+	assert.Equal(t, exportedPolicy, importedPolicy)
+}
+
+// Converts Throttling policy struct to map
+func ThrottlePolicyStructToMap(policy interface{}) (map[string]interface{}, error) {
+	var throttlePolicy map[string]interface{}
+	marshalled, _ := json.Marshal(policy)
+	err := json.Unmarshal(marshalled, &throttlePolicy)
+	return throttlePolicy, err
+}
+
+// Cleanup func for exported throttling policy file
+func RemoveExportedThrottlingPolicyFile(t *testing.T, file string) {
+	t.Log("base.RemoveExportedThrottlingPolicyFile() - file path:", file)
+	if _, err := os.Stat(file); err == nil {
+		err := os.Remove(file)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+}
+
+// Validates whether the throttling policy list is complete
+func ValidateThrottlePoliciesList(t *testing.T, jsonPretty bool, args *ThrottlePolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	base.WaitForIndexing()
+
+	output, _ := listThrottlePolicies(t, jsonPretty, args)
+
+	throttlePoliciesList := args.SrcAPIM.GetThrottlePolicies(t, args.CtlUser.Username, args.CtlUser.Password)
+
+	ValidateListThrottlePoliciesEqual(t, output, throttlePoliciesList)
+}
+
+// get the Throttle policy list apictl output
+func listThrottlePolicies(t *testing.T, JsonPretty bool, args *ThrottlePolicyImportExportTestArgs) (string, error) {
+	var output string
+	var err error
+	if JsonPretty {
+		output, err = base.Execute(t, "get", "policies", "rate-limiting", "-e", args.SrcAPIM.EnvName, "--format", "\"{{ jsonPretty . }}\"", "-k", "--verbose")
+	} else {
+		output, err = base.Execute(t, "get", "policies", "rate-limiting", "-e", args.SrcAPIM.EnvName, "-k", "--verbose")
+	}
+	return output, err
+}
+
+// Checks whether apictl output contains all the available throttle policy UUIDs
+func ValidateListThrottlePoliciesEqual(t *testing.T, throttlePoliciesListOutput string, throttlePoliciesList *utils.ThrottlingPoliciesDetailsList) {
+	unmatchedCount := throttlePoliciesList.Count
+	for _, policy := range throttlePoliciesList.List {
+		// If the output string contains the same Policy ID, then decrement the count
+		assert.Truef(t, strings.Contains(throttlePoliciesListOutput, policy.Uuid), "throttlePoliciesListFromCtl: "+throttlePoliciesListOutput+
```

Comment:
Why are we using two assertion for the same validation? Line 238 & 243

Reply:
Line 238 validates whether the output contains all the policy ids and while doing that we can check whether the count also correct by decrementing count until it's zero which is checked in line 243